frontend/src/components/Banners/SearchBanner/index.js

```diff
@@ -1,35 +1,35 @@
-import React from "react";

```

Comment:
Please, read once again this comment, I think you misunderstood it

<img width="795" alt="Screenshot 2021-03-11 at 18 01 45" src="https://user-images.githubusercontent.com/25443181/110833018-daf19180-8293-11eb-8372-eeff2ad98b52.png">

Reply:
I have not misunderstood, I renamed all my components to index.js and I am importing SearchBanner and all other components like 
import SearchBanner from "../../Banners/SearchBanner"